src/index.html

```diff
@@ -0,0 +1,227 @@
+                   <img class="promo__2badges-img" src="/src/image/price.png" alt="Prise"/>
+                   <div class="promo__2badges-right">
+                       <div class="promo__2badges-desc">Instant price</div>
+                       <div class="promo__2badges-title">3.5 ETH</div>
+                   </div>
+                </div>
+            </div>
+            <div class="promo__bid">
+                <div class="promo__bid--top">
+                    <div class="promo__bid--top-desc">Current Bid</div>
+                    <h2 class="promo__bid--top-title">1.00 ETH</h2>
+                    <div class="promo__bid--top-price">$3,618.36</div>
+                    <div class="promo__bid--top-auction">Auction ending in</div>
+                </div>
+                <div class="promo__bid--timer">
+                    <div class="promo__bid--timer-numbers">
+                        <div class="promo__bid--timer-number">19</div>
+                        <div class="promo__bid--timer-number">24</div>
+                        <div class="promo__bid--timer-number">19</div>
+                    </div>
+                    <div class="promo__bid--timer-letters">
+                        <div class="promo__bid--timer-letter">Hrs</div>
+                        <div class="promo__bid--timer-letter">mins</div>
+                        <div class="promo__bid--timer-letter">secs</div>
+                    </div>
+                </div>
+            </div>
+            <div class="promo__buttons">
+                <div class="promo__button1 button button--blue">Place a bid</div>
+                <div class="promo__button2 button">View item</div>
+            </div>
+            <div class="promo__arrows">
+                <button class="promo-arrow button--arrow-left" title="Left arrow"></button>
+                <button class="promo-arrow button--arrow-right" title="Right arrow"></button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="future">
+    <div class="future__wrap">
+        <div class="future__1column">
+            <img class="future__1column--image" src="/src/image/02.png" alt="Picture" />
+            <div class="future__1column--stroke">
+                <div class="future__1column--stroke-left">
+                <img class="future__1column--img" src="/src/image/user 48x48.png" alt="Picture" />
+                <div class="future__1column--text">
+                    <div class="future__1column--text-title">The future of ETH®</div>
+                    <div class="future__1column--text-desc">18 in stock</div>
+                </div>
+                </div>
+                <div class="future__1column--info">
+                    <div class="future__1column--info-desc">Highest bid</div>
+                    <div class="future__1column--info-price">1.125 ETH</div>
+                </div>
+            </div>
+        </div>
+        <div class="future__2column">
+            <div class="future__2column--card">
```

Comment:
Better to move to separate entity card and style with modifiers e.g `--large` `--small`